kodadot/nft-gallery
# It's normal

- temp refactor
- feat: gains for user
- action test
- add ShoppingActions as private value
- refactor: subsquid for ProfileDetail
- code refactor
- fix user column widths
- refactor
- remove unused test packages
- refactor: delete old error handling methods
- feat: implement new generic error page
- refactor: resolve via correct error handler
- [skip netlify] Update Bumped minor version of packages
- remove_address_to_clipboard_copy
- Replace rubick@004 queries to use rubick@006
- Update components/shared/format/Identity.vue
- refactor: remove redundant check
- fix: hide burned nft in gains tab
- fix: Sort visited NFTs by time user has visited them
- fix: show burned nft
- user profil with history events
- delete console.log
- feat: change client and import address
- fix: process data
- [skip netlify] Update dependency vitest to ^0.12.0
- feat: remap sortby keywords
- feat: update imported graphql
- fix: merge conflict
- deleting service-worker.js from post january migration
- Add on series price per buy
- Sometimes historically visited NFTs component does not appears
- Revert "Sometimes historically visited NFTs component does not appears"
- show visited nft that has not been listed
- feat: relocate
- fix: replace gmt with umt in history date column
- feat: also show path that caused error
- fix:query with LIMIT and OFFSET with pagination
- use PrefixMixin
- Filter on top of collection detail show only collected
- use bulma mixins instead of media query
- :zap: instant link to create on-chain identity from profile
- remove average price sort
- add event gragments
- Update layouts/error.vue
- profil_activity component
- fix: search broken
- fix query events + watch active tab
- refactor: use tolocalestring instead
- feat: Addressable spotlight
- #2819 Show collection first, gallery second in explore preference
- [skip netlify] Update Bumped minor version of packages
- :bug: httpcat is parsed as string not as variable
- remove unused constants, variables, args, functions
- justify translation
- fix: show owner switch
- no create function
- fix listed count
- unused vars
- fix UI and comments
- 🔄 wallet address
- Update components/rmrk/Profile/ProfileDetail.vue
- added monthly active contributors
- fix: remove useless comment
- fix: and
- fix: empty key at vue component
- refactor: history on profile
- [skip netlify] Update Bumped minor version of packages
- refactor: using fragment
- fix: searchbar issue
- leaderboardUpdate_05-12-2022
- burnRateUpdate_05-12-2022
- flipper default link to user gains
- fix: user link
- fix more linter warnings
- 🌍 remove unused translation
- [skip netlify] Update dependency @ramp-network/ramp-instant-sdk to ^3.1.1
- 🧹 cleaning translation
- 🧹 moar cleaning
- delete extended value
- fix: name issue and listed filter
- refactor: update field name
- fix: statemin
- fix: westmint
- fix: emoteCount
- fix: collectionMeta
- ⚒️ teleport page name lang
- total_emotes_count_feat
- fix: miss unlist event at history
- handle_no_emotes
- fix: Relocate first page query in searchbar
- [skip netlify] Update dependency vitest to ^0.12.6
- feat: scroll to top button
- addded snapshot from repo trends:)
- fix: remove suggestion query for other chains
- feat: Show whale collected nfts from particular creator
- correct translation text
- fix: remove unused import
- feat: passion feed request
- feat: passion feed switch ui
- feat: router query
- assign_bot_update
- refactor history component
- feat: latest sales with pasison filter
- refactor: landing page
- feat: LastestSales and NewestList
- fix: relocate general graphql
- fix: rename fragment
- fix: add currentOwner
- ui refactor
- history component refactor v2
- fix: profile history amount issue
- review + clean & facto
- subquery use fragment
- UI justify
- refactor: history component in gallery details page
- fix: using this.client
- feat: page for sale engagement
- fix: sold tab can not search
- feat: saels feed query
- feat: init page for sales
- refactor: update time format
- handle search on query
- simple component data
- refactor history v3
- refactor: using basic image
- feat: popup for larger nft image
- feat: update basic popup
- check redos on rmrk/utils.ts => change function to rr.exec()
- NftUtils.ts => change function to rr.exec()
- extension.ts => check with boolean
- ipfs.ts => change regex because doesnt work + Opti with function
- 🧹 deepsource issues
- sales view for a creator
- unused vars
- fix: typo
- feat: rename to RowSales and add type file
- fix: issues by request
- [skip ci] updated code diagram
- fix
- refactor history component
- try opti query
- used unwrap minimark
- unused vars
- opti subquery performance
- typo and clean
- pagination right
- clean
- eslint format
- #3035 Column overflow issue
- fix: based on request
- clean: remove unused code and reuse minimark
- fix: request
- refactor: add UNLIST event type
- stats column
- [skip netlify] Update Bumped minor version of packages
- chore: event type
- chore: add blank line
- [skip netlify] Update dependency vitest to ^0.12.9
- feat: add qr code page
- remove cypress
- unused vars
- deepscan
- deep
- deep
- fix: Profile wont load when you go back
- ⚒ unwrap warning & pnpm outdated
- refactor: remove something unused
- feat: update events on demand
- fix: graphql nftEntities to nft
- eslint types
- deepsource
- scrollable series insight table
- [skip netlify] Update Bumped minor version of packages
- feat: Add difference from previous trade into history
- percentage column visible
- link qr code to footer
- [skip netlify] Update dependency chart.js to ^3.8.0
- fix total sold
- fix: type check
- feat(profile): fetch total numer of lots of tabs
- feat(profile): hide tabs if it is 0
- fix: based on request.
- fix: remove string and using fetch

**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type


### What's new?


### Before submitting Pull Request, please make sure:


### Optional


### Had issue bounty label?


### Community participation


### Screenshot
